### PR TITLE
[ASL-4413] HBA upload functionality in PPL transfer journeys

### PR DESCRIPTION
**Pull request by edemirbag**
Latest functional and content changes towards ASL-4413, including those cherry picked from commit b77a5381e9aaf9badf92f9cb85a62eb96b8c0706

### CODE REVIEW

File: pages/task/read/views/confirm-hba.jsx [hunk 1]
```diff
@@ -91,8 +24,27 @@ const ConfirmHba = ({ establishment, licenceHolder, hba, task }) => {
...
+        <p>
+          <strong>
+            <Snippet>fields.applicant.label</Snippet>
+          </strong>
+          <br />
+          {task.data.profile.name}
+        </p>
+        <p>
+          <strong>
+            <Snippet>fields.proposedApplicant.label</Snippet>
```
Comment (farrukhishaq) [inline pages/task/read/views/confirm-hba.jsx:43]: The modification I made.

File: pages/task/read/views/confirm-hba.jsx [hunk 2]
```diff
@@ -91,8 +24,27 @@ const ConfirmHba = ({ establishment, licenceHolder, hba, task }) => {
...
+          </strong>
+          <br />
+          {establishment.name}
+        </p>
+        <p>
+          <strong>
+            <Snippet>fields.applicant.label</Snippet>
+          </strong>
+          <br />
+          {task.data.profile.name}
```
Comment (farrukhishaq) [inline pages/task/read/views/confirm-hba.jsx:39]: current license holder

File: pages/success/content/index.js [hunk 3]
```diff
@@ -105,21 +105,21 @@ module.exports = {
...
     whatNext: {
-      body: `This request has been approved.`,
+      body: ``,
       internal: 'The relevant people have been emailed.',
       external: ''
     },
     taskLink: {
-      before: 'You can ',
-      linkText: 'view the history of this request.'
+      before: '',
```
Comment (farrukhishaq) [inline pages/success/content/index.js:113]: removing `You can` is this intentional?

File: pages/task/schema/view.js [hunk 4]
```diff
@@ -25,6 +40,8 @@ module.exports = task => {
       };
     });
 
+  console.log(options);
```
Comment (farrukhishaq) [inline pages/task/schema/view.js:43]: remove this `console.log`

File: pages/success/content/index.js (hunk 3 above)
Comment (edemirbag) [reply pages/success/content/index.js:113]: yes, a copy change requested by UCD team

File: pages/task/read/content/confirm-hba.js [hunk 5]
```diff
@@ -10,8 +10,17 @@ module.exports = merge({}, baseContent, {
...
+    },
+    proposedEstablishment: {
+      label: 'Proposed establishment:'
+    },
     applicant: {
-      label: 'Applicant:'
+      label: 'PPL holder:'
+    },
+    proposedApplicant: {
+      label: 'Proposed PPL licence holder:'
```
Comment (farrukhishaq) [inline pages/task/read/content/confirm-hba.js:23]: @edemirbag this is the only change, and name used.

Comment (edemirbag) [reply pages/task/read/content/confirm-hba.js:23]: noted - will be easier to discuss this over a call

Comment (edemirbag) [reply pages/task/read/content/confirm-hba.js:23]: For reference:

- https://github.com/UKHomeOffice/asl-pages/commit/3473b29f371490f7b45aa4da51cdef5d79da873b
- https://github.com/UKHomeOffice/asl-pages/commit/4a5ad8ce89a52f063b8e47d9a72baa09a6fe0058

**Review by farrukhishaq — APPROVED** [review]

**Pull request merged**